core/src/components/range/range.md.scss

```diff
@@ -63,6 +64,32 @@
   transition-timing-function: ease;
 
   z-index: 2;
+
+  &::before {
+    @include border-radius(50%);
+    @include position(null, null, null, 0);
+
+    @include rtl() {
+      /* stylelint-disable-next-line property-disallowed-list */
+      left: unset;
+    }
+
+    position: absolute;
+
+    width: var(--knob-size);
+    height: var(--knob-size);
+
+    transform: scale(1);
+
+    // Value from Material Design components for the web
+    transition: 0.267s cubic-bezier(0, 0, 0.58, 1);
+
+    background: var(--knob-background);
+
+    content: "";
+    opacity: 0.3;
```

Comment:
Thoughts on starting the opacity at whatever the hover + focus opacity is? Otherwise you get a strange fade effect when hovering over the range (range is enlarged + animation is slowed down for emphasis)

https://user-images.githubusercontent.com/2721089/189433236-13527017-1c5a-4364-b51f-fedc1c8e281c.mov

Alternatively, this issue may less noticeable if the hover opacity is the same as the focus opacity.